grakn-test/test-integration/src/test/java/ai/grakn/test/engine/GrpcServerIT.java

```diff
@@ -112,6 +112,19 @@ public void whenPuttingEntityType_EnsureItIsAdded(){
         }
     }
 
+    @Test
+    public void whenGettingEntityType_EnsureItIsReturned(){
+        String label = "Oliver";
+        try (GraknTx tx = localSession.open(GraknTxType.WRITE)) {
+            tx.putEntityType(label);
+            tx.commit();
+        }
+
+        try (GraknTx tx = remoteSession.open(GraknTxType.WRITE)){
+            assertNotNull(tx.getEntityType(label));
```

Comment:
We could assert the label is correct to be more sure we have the right thing... but I guess that has to wait for my PR #2656